tests/integration_tests/test_sequence_features.py

```diff
@@ -79,34 +77,29 @@ def setup_model_scaffolding(raw_df, input_features, output_features):
         yield model, batcher
 
 
-# TODO(#1333): refactor test once torch sequence generator work is complete
+# TODO(#1333): Refactor this test once torch sequence generator work is complete.
+# - Tests may be covered by other smaller scoped unit tests.
 #
 # tests output feature sequence with `Generator` decoder
 # pytest parameters
 #   dec_cell_type: decoder cell type
-#   dec_attention: decoder's attention mechanism
-#   dec_beam_width: decoder's beam search width
 #   combiner_output_shapes: is a 2-tuple specifies the possible types of
 #     tensors that the combiner may generate for sequences.
 #     combiner_output_shapes[0]: specifies shape for hidden key
 #     combiner_output_shapes[1]: is either None or 1 or 2-tuple representing
 #       the encoder_output_state key. None: no encoder_output_state key,
 #       1-tuple: generate tf.Tensor, 2-tuple: generate list with 2 tf.Tensors
-#
-@pytest.mark.skip(reason="Issue #1333: Sequence output generation.")
-@pytest.mark.parametrize("dec_num_layers", [1, 2])
-@pytest.mark.parametrize("dec_beam_width", [1, 2])
-@pytest.mark.parametrize("dec_attention", ["bahdanau", "luong", None])
 @pytest.mark.parametrize("dec_cell_type", ["lstm", "rnn", "gru"])
 @pytest.mark.parametrize(
     "combiner_output_shapes",
-    [((128, 10, 8), None), ((128, 10, 32), None), ((128, 10, 8), ((128, 8), (128, 8))), ((128, 10, 8), ((128, 8),))],
+    [
+        ((128, 10, TEST_STATE_SIZE), None),
+        ((128, 10, TEST_STATE_SIZE), ((128, TEST_STATE_SIZE), (128, TEST_STATE_SIZE))),
+        ((128, 10, TEST_STATE_SIZE), ((128, TEST_STATE_SIZE),)),
+    ],
 )
 def test_sequence_decoders(
     dec_cell_type,
-    dec_attention,
-    dec_beam_width,
-    dec_num_layers,
```

Comment:
add todo to reintroduce these?

Reply:
Done. This test really should belong in test_sequence_decoders.py unit tests, and tests should certainly be added when reintroduce beam search, multi-layers, and attention.